docs/plugins/add-to-directory.md

```diff
@@ -25,3 +25,14 @@ iconUrl: # Used as the src attribute for your logo.
 npmPackageName: # Your npm package name E.g. '@backstage/plugin-<etc>' quotes are required
 addedDate: # The date plugin added to directory E.g. '2022-10-01' quotes are required
 ```
+
+## Submission Tips
+
+Here are a few tips to help speed up the review process when you submit your plugin:
+
+- For any icon that you use make sure you have the proper rights to use it.
```

Comment:
on the icon: I believe the example for the `iconUrl` above is misleading. When using a relative path to static the `iconUrl` should start with a `/`, right?

Reply:
Good call, this actually just came up: https://github.com/backstage/backstage/pull/21524#discussion_r1404407622

I've added the `/` 👍